apps/website/src/pages/institute/pixels.tsx

```diff
@@ -101,16 +113,13 @@ const InstitutePixelsPage: NextPage = () => {
     });
   }, []);
 
-  const handleScrollbarChange = useCallback(
-    (e: ChangeEvent<HTMLInputElement>) => {
-      const progress = parseFloat(e.target.value);
-      setScrollProgress(progress);
-      if (!fullscreenRef.current) return;
-      const { scrollWidth, clientWidth } = fullscreenRef.current;
-      fullscreenRef.current.scrollLeft = (scrollWidth - clientWidth) * progress;
-    },
-    [],
-  );
+  const handleScrollbarChange = useCallback((e: ChangeEvent) => {
+    const progress = parseFloat(e.target.value);
+    setScrollProgress(progress);
+    if (!fullscreenRef.current) return;
+    const { scrollWidth, clientWidth } = fullscreenRef.current;
+    fullscreenRef.current.scrollLeft = (scrollWidth - clientWidth) * progress;
+  }, []);
```

Comment:
Why the unrelated changes here?

```suggestion
  const handleScrollbarChange = useCallback(
    (e: ChangeEvent<HTMLInputElement>) => {
      const progress = parseFloat(e.target.value);
      setScrollProgress(progress);
      if (!fullscreenRef.current) return;
      const { scrollWidth, clientWidth } = fullscreenRef.current;
      fullscreenRef.current.scrollLeft = (scrollWidth - clientWidth) * progress;
    },
    [],
  );
```